_topics/en/erlay.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Erlay
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Bandwidth Reduction
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Erlay** is proposal to use set reconciliation between Bitcoin full
+  nodes as part of relaying unconfirmed transactions.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  The currently-used Bitcoin gossip protocol expects each node to
+  advertise every new transaction to all of its peers.  However, peers
+  only need to learn about the transaction from any one of their peers,
+  so a significant amount of bandwidth is wasted on redundant
+  announcements.  Set reconciliation using [libminisketch][] would
```

Comment:
As above, I think we should mention that erlay has two phases: fanout and reconciliation.

Reply:
Done